vagrant/installer.sh

```diff
@@ -25,8 +25,20 @@ chmod og-wx ~/.ssh/authorized_keys
 
 # Install dependencies
 sudo swapoff -a
-curl -sL https://bootstrap.pypa.io/get-pip.py | sudo python
-sudo -E pip install ansible
+if [ -e /etc/fstab ]; then
+    sudo sed -i '/ swap / s/^/#/' /etc/fstab
+fi
+# shellcheck disable=SC1091
+source /etc/os-release || source /usr/lib/os-release
+case ${ID,,} in
+    rhel|centos|fedora)
+        curl -sL https://bootstrap.pypa.io/get-pip.py | sudo python
```

Comment:
Doesn't `pip` or `ansible` come packaged by the distros? We would get a known version and safer installation this way.

Reply:
Ok, I'm going to fix the installation of ansible to have better control

Reply:
Fixed on a942dfa05db723a84f821c24109e0f14ae1b9683